acme/acme.go

```diff
@@ -549,16 +606,36 @@ func (a *ACME) LoadCertificateForDomains(domains []string) {
 		} else {
 			domain = Domain{Main: domains[0]}
 		}
+
 		if _, exists := account.DomainsCertificate.exists(domain); exists {
-			// domain already exists
+			log.Debugf("ACME Certificate for domain %v exists.", domain)
 			return
 		}
+
+		// Check if our domains are matching our ingoreFilters
+		if len(a.IgnoreFilters) > 0 {
```

Comment:
Is it possible to put all the `IgnoreFilters` code into a function? It seems to like https://github.com/containous/traefik/pull/1260/files#diff-faa8f9dd50dafacfcb04a8203ce3deebR542 :)